cmd/generate-config/system_vars_def.toml

```diff
@@ -327,6 +327,36 @@ values = ["true", "false"]
 comment = "default is true. if true, enable metric at booting"
 update-mode = "dynamic"
 
+[[parameter]]
+name = "enableTrace"
+scope = ["global"]
+access = ["file"]
+type = "bool"
+domain-type = "set"
+values = ["true", "false"]
+comment = "default is true. if true, enable trace at booting"
+update-mode = "dynamic"
+
+[[parameter]]
+name = "traceBatchProcessor"
+scope = ["global"]
+access = ["file"]
+type = "string"
+domain-type = "set"
+values = ["InternalExecutor", "FileService"]
+comment = "default is InternalExecutor. if InternalExecutor, use internal sql executor, FileService will implement soon."
+update-mode = "dynamic"
+
+[[parameter]]
+name = "enableTraceDebug"
+scope = ["global"]
+access = ["file"]
+type = "bool"
+domain-type = "set"
+values = ["false", "true"]
+comment = "default is false. With true, system will check all the children span is ended, which belong to the closing span."
```

Comment:
What is a span?    I guess I will read it later in the commit.   However, the name of the parameter, enableTraceDebug, and the comment, does not seem to be the same thing.